packages/core/src/Step.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Flex from './Flex'
+import Text from './Text'
+import Icon from 'pcln-icons'
+
+const StyledFlex = styled(Flex)`
+  cursor: ${props => (props.onClick ? 'pointer' : 'default')};
+`
+
+function Step({ className, active, completed, children, onClick }) {
+  const color = active || completed ? 'primary' : 'text.light'
+
+  return (
+    <StyledFlex className={className} alignItems="center" onClick={onClick}>
+      {completed && (
+        <Icon name="check" color="primary" size={16} mr={1} legacy={false} />
```

Comment:
could you import CheckIcon instead, please?

Reply:
this will force every icon into consuming bundles, even if they have removed all uses of the bare Icon

Reply:
Yes that's a problem. However, I tried using the specific icon but that cause an issue in the docs project where it couldn't find the `css` function from styled components which is used to handle the palette colors. Adam and I looked into this thoroughly but could not resolve the issue and said that this is a good enough approach for now and that I should add a tech debt issue once this PR is merged to look into the issue further.

Reply:
i suppose that would be ok if there is a commitment to fast follow on it. but i'm really hesitant to allow a single component to force all consumers of DS to bundle every icon for an issue you say is limited to the current docsite.

we should discuss in Friday's WG

Reply:
is the fast follow ticketed and will be worked in the short term? i'd rather omit this from the docsite than force bundling of all the icons if not

Reply:
Agreed, I think that's a better short-term solution, I'll make the change.